### PR TITLE
Add Bulgarian translation

### PR DESCRIPTION
**Pull request by Keithbm1**
- [x] I created a new folder `i18n/bg` for my translated files
- [x] I translated `metadata.yml` file
- [x] I translated `white-paper.md` file

### CODE REVIEW

File: i18n/bg/metadata.yml
Comment (mfts) [inline i18n/bg/metadata.yml:78]:
```suggestion
- \fancyfoot[L]{v1.0.3+bg}   # expand these variables!
```

this needs to be properly aligned

**Comment by mfts** [comment]
@Xercesblu3 please run workflow

File: i18n/bg/metadata.yml
Comment (Keithbm1) [reply i18n/bg/metadata.yml]: @mfts Hello, I copied the white-paper translation from my other PR the i18n/bg folder.
So now, I can close the other PR#83?

Comment (mfts) [reply i18n/bg/metadata.yml]: @Keithbm1 Thank you and yes you can close #83 now

File: i18n/bg/metadata.yml
Comment (mfts) [inline i18n/bg/metadata.yml:3]:
```suggestion
  Създаването на отворен, публичен и стабилен регистър за целия софтуер с отворен код 
```

Please properly indent as this is YAML and correct indents are important

File: i18n/bg/metadata.yml
Comment (mfts) [inline i18n/bg/metadata.yml:82]:
```suggestion
- Svetlana Fyodorovna Dulapchi
```
Improved indention

File: i18n/bg/white-paper.md
Comment (mfts) [inline i18n/bg/white-paper.md:277]: Please change the translation of the reference key back from `фиг` back to `fig` 

And please instead translate: 

- "Dependencies analysis diagram."
- "Steeping rewards distribution across dependencies."

**Review by mfts — CHANGES_REQUESTED** [review]
Minor details that need to be fixed. Otherwise looks really good and almost ready to be merged

File: i18n/bg/white-paper.md
Comment (Keithbm1) [reply i18n/bg/white-paper.md]: @mfts changes Updated. It will take me a little time to re-check the translation for possible spelling mistakes in Bulgarian

Comment (Keithbm1) [reply i18n/bg/white-paper.md]: @mfts Hello, are there more changes that I have to fix up? 
I have re-checked the translation for possible spelling mistakes in Bulgarian. What are the next steps? Thank's

Comment (mfts) [reply i18n/bg/white-paper.md]: @Keithbm1 thank you for your effort. Everything works now and it's ready for publication!

**Review by mfts — APPROVED** [review]
Compiles locally and is ready to merge, @Xercesblu3 

We could wait for @jhheider to fix the CI but I tested it locally (which works) and it's approved to merge from my end.

**Comment by Xercesblu3** [comment]
Will wait for Jacob'a fix before merging.

**Comment by mfts** [comment]
@Xercesblu3 bulgarian translation is ready for merge

**Pull request merged**
